corehq/apps/campaign/models.py

```diff
@@ -13,39 +14,63 @@ class Meta:
         app_label = 'campaign'
 
 
-class DashboardMap(models.Model):
+class DashboardTab(models.TextChoices):
+    """
+    The tab on which a dashboard widget is displayed
+    """
+    CASES = 'cases', _('Cases')
+    MOBILE_WORKERS = 'mobile_workers', _('Mobile Workers')
+
+
+class DashboardWidgetBase(models.Model):
+    """
+    Base class for dashboard widgets
+    """
+    title = models.CharField(max_length=255, blank=True, null=True)
+    description = models.CharField(max_length=255, blank=True, null=True)
```

Comment:
I believe this would better be a `TextField`. I can imagine this getting longer than 255.